### PR TITLE
Backport IAM FAQ + Security pages changes

### PR DESCRIPTION
**Pull request by viorelaioia**
@gdestuynder , r?

### CODE REVIEW

File: Security.mediawiki [hunk 1]
```diff
@@ -16,9 +16,11 @@ Details on the way we classify security bugs can be [[Security Severity Ratings|
 == Security at Mozilla ==
```
Comment (gdestuynder) [inline Security.mediawiki]: looks like the previous change on this page wasn't sync'd, if you edit from git locally try: ```git pull --rebase <origin|upstream|however your upstream is called>```

File: IAM/Frequently_asked_questions.mediawiki [hunk 2]
```diff
@@ -81,6 +81,32 @@ Two-factor authentication can be configured by using a Time-based One-Time Passw
...
+   [[File:Github_-_set_up_2fa.png|400px]]
+4.  On the Two-factor authentication page, click Set up using an app.
+   [[File:Github_-_set_up_using_an_app.png|400px]]
+5.  To enable two-factor authentication you must save your two-factor recovery codes in a safe place, such as a password manager, by clicking Download, Print, or Copy. Your recovery codes will help get you back into your account if you've lost access. After saving your codes, click Next.
+   [[File:Github_-_recovery.png|450px]]
+6.  Get the secret key for your duo setup by either scanning the barcode (the URI generated should have the "secret" parameter) or by clicking the "enter this text code" link. Also, save this secret in a file. 
+   [[File:Github_-_enter_text_code.png|450px]]
+   [[File:Github_-_2fa_secret.png|300px]]
+7.  Copy the 2-factor secret.<br>
+8.  Navigate to https://firekey.org/ and add a new account. Enter a site name and paste the secret in "secret key" field, then click "Add".
```
Comment (gdestuynder) [inline IAM/Frequently_asked_questions.mediawiki]: I don't think we can recommend this as this goes against our security fundamentals, effectively storing both factors on the same devices: https://infosec.mozilla.org/fundamentals/rationales.html#mfa

File: Security.mediawiki [hunk 3]
```diff
@@ -16,9 +16,11 @@ Details on the way we classify security bugs can be [[Security Severity Ratings|
...
 === Who are we? ===
```
Comment (gdestuynder) [inline Security.mediawiki]: this commit needs to be removed

**Comment by gdestuynder** [comment]
cc @gene1wood @jeffbryner

**Comment by hmitsch** [comment]
@gdestuynder there are two problems we try to solve with this FAQ entry.

1. **Mozillians in India**
We have a number of NDA'd Volunteers in India. GitHub does not support SMS authentication in India. Some of our contributors do not have smartphones. This was the reason people were looking for a 2FA solution which does not require a mobile phone. 
   * Do you have any alternative suggestion for this scenario? 
   * Do you think it's possible to ask GitHub to add India to the list of countries for SMS authentication?
1. **People not wanting to use Google Authenticator**
If I remember correctly, some people do not want to use the Google Authenticator or Duo apps.
   * Do you know of any other open source apps we can recommend in these cases?

**Comment by hmitsch** [comment]
On a side-note, looking at https://infosec.mozilla.org/fundamentals/rationales.html#mfa: 

Does this mean I am not allowed to use a **password manager** (such as 1Password) for my Mozilla LDAP password on my **mobile phone** because it is also **my Duo MFA device**?

File: IAM/Frequently_asked_questions.mediawiki (hunk 2 above)
Comment (gene1wood) [reply IAM/Frequently_asked_questions.mediawiki]: To be explicit, the comparisons would be

# Comparisons

## User logs in on computer and uses TOTP code from app on phone

Not a problem, the second factor secret is not stored on the same device as the one being used to login (it's stored in the phone's sequestered storage for the app)

## User logs in on computer and uses TOTP code from firekey

Not a problem, the second factor secret is not stored on the same device as the one being used to login (it's stored in the phone's browser's localStorage)

## User logs in on phone and uses TOTP code from app on phone

The problem here is that the phone has access to both the password and the second factor secret. If the phone is compromised then the account is compromised.

## User logs in on phone and uses TOTP code from firekey

The problem here is that the phone has access to both the password and the second factor secret. If the phone is compromised then the account is compromised.

# Tangential issues

It's worth noting that the second factor secret is stored in different places with potentially different security profiles if you're talking about an app or a website

@gdestuynder can speak with more authority about where the duo or google authenticator apps store the secret and what the mobile OS security model (assuming it's not rooted) means for the security around that secret.

For Firekey, it's storing the secret in the browser's localStorage which has a different set of attacks.

**Comment by gene1wood** [comment]
@hmitsch 

> People not wanting to use Google Authenticator

Ya, Duo's app would work. I use [FreeOTP](https://freeotp.github.io/)

**Comment by viorelaioia** [comment]
Closing this, as the changes were reverted in the wiki page.

**Pull request closed without merging**